forks/frames/pom.xml

```diff
@@ -164,7 +164,6 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.4.2</version>
```

Comment:
It's too old to understand the `@{...}` syntax, which is needed because the JaCoCo Maven plugin sets the property dynamically.